wp-content/themes/twentytwenty-child/settings-student.js

```diff
@@ -57,4 +57,23 @@ jQuery(document).ready(function($){
         });
     });
 
-});
\ No newline at end of file
+    var display = Number($(".infinite-scroll").attr("value"));
+    $(window).scroll(function(e){
+        if( $(window).scrollTop() + $(window).height() >= $(document).height() ) {
```

Comment:
You are running `$(window)` multiple times. Also pretty sure you don't need to run $() to get the window or document element.